xml/vt_docker_quick.xml

```diff
@@ -328,7 +328,7 @@
    </para>
 <!--taroth 2015-07-02: http://doccomments.provo.novell.com/comments/28724 -->
    <procedure>
-    <title>Enabling the Container Module</title>
+    <title>Enabling the Container Module - interactive way</title>
```

Comment:
I could try to make the point that ncurses YaST is actually half-graphical, but I won't. It does run on the command line after all.

Maybe `Enabling the Controller Module Using &yast;` vs. `Enabling the Controller Module Directly with SUSEConnect`? That is quite long, though.